.github/workflows/build.yaml

```diff
@@ -201,37 +205,55 @@ jobs:
+
+      - name: Checkout
+        uses: actions/checkout@v3
+
+      - name: Login to the ${{ env.DOCKER_REGISTRY }} Container Registry
+        uses: docker/login-action@v2
         with:
-          name: nodeenv-docker-image
-          path: build/          
-      - name: Push to registry
-        run: |
-          set -xev
-          wget -qO "$PWD/manifest-tool" https://github.com/estesp/manifest-tool/releases/download/v1.0.0/manifest-tool-linux-amd64
-          chmod +x ./manifest-tool
+          registry: ${{ env.DOCKER_REGISTRY }}
+          username: ${{ env.DOCKER_REGISTRY == 'docker.io' && secrets.DOCKERHUB_USERNAME || github.actor }}
+          password: ${{ env.DOCKER_REGISTRY == 'docker.io' && secrets.DOCKERHUB_TOKEN    || secrets.GITHUB_TOKEN }}
 
-          echo "Version strings are ${PACKAGE_VERSION} ${MINOR_PACKAGE_VERSION}"
+      - name: Docker meta
+        id: meta
+        uses: docker/metadata-action@v4
+        with:
+          images: ${{ env.DOCKER_REGISTRY }}/${{ github.repository_owner }}/fabric-nodeenv
+          tags: |
+            type=semver,pattern={{version}}
+            type=semver,pattern={{major}}.{{minor}}
+            type=semver,pattern={{major}}.{{minor}}.{{patch}}
 
-          docker image load --input build/fabric-javaenv.tar.gz
-          docker images
-          docker login ${DOCKER_REGISTRY_URL} --username=${DOCKER_REGISTRY_USERNAME} --password=${DOCKER_REGISTRY_PASSWORD}
-          echo "Logged in to docker registry"
-          # tag javaenv image to PACKAGE_VERSION
-          docker tag hyperledger/fabric-javaenv hyperledger/fabric-nodeenv:amd64-${ PACKAGE_VERSION }
-          # push javaenv to repository
-          docker push hyperledger/fabric-javaenv:amd64-${ PACKAGE_VERSION }
-          ./manifest-tool push from-args --platforms linux/amd64 --template "hyperledger/fabric-nodeenv:amd64-${ PACKAGE_VERSION }" --target "hyperledger/fabric-nodeenv:${ PACKAGE_VERSION }"
-          ./manifest-tool push from-args --platforms linux/amd64 --template "hyperledger/fabric-nodeenv:amd64-${ PACKAGE_VERSION }" --target "hyperledger/fabric-nodeenv:${ MINOR_PACKAGE_VERSION }"
-        env:
-          DOCKER_REGISTRY_USERNAME:  ${{ secrets.DOCKERHUB_USERNAME }}
-          DOCKER_REGISTRY_PASSWORD:  ${{ secrets.DOCKERHUB_TOKEN }}      
-          PACAKGE_VERSION: ${{  needs.setup.outputs.PACKAGE_VERSION }}
-          MINOR_PACKAGE_VERSION: ${{ needs.setup.outputs.MINOR_PACKAGE_VERSION }}
+      - name: Build and push ${{ matrix.COMPONENT }} Image
+        id: push
+        uses: docker/build-push-action@v3
+        with:
+          platforms: linux/amd64,linux/arm64
+          file: docker/fabric-nodeenv/Dockerfile
+          context: docker/fabric-nodeenv
+          tags: ${{ steps.meta.outputs.tags }}
+          push: ${{ github.event_name != 'pull_request' }}
+          labels: ${{ steps.meta.outputs.labels }}
+          build-args: |
+            NODE_VER=${{ env.NODE_VER }}
```

Comment:
Oh this is the arg to the dockerfile for the version of node to push in.... can we alter the name so to reflect that?